lib/gera/binance_fetcher.rb

```diff
@@ -19,13 +19,22 @@ def perform
         cur_to = find_cur_to(symbol, cur_from)
         next unless cur_to
 
+        next if price_is_missed?(rate: rate)
+
         pair = CurrencyPair.new(cur_from: cur_from, cur_to: cur_to)
         memo[pair] = rate
       end
     end
 
     private
 
+    # NOTE: for some pairs price is "0.00000000"
+    def price_is_missed?(rate:)
+      return true if rate['askPrice'].to_f == 0.0 || rate['bidPrice'].to_f == 0.0
+
+      false
```

Comment:
в одну строку `rate['askPrice'].to_f == 0.0 || rate['bidPrice'].to_f == 0.0`

сравниение флоатов всегда плохая идея. Почему вообще приходит рейт 0? это какая-то фигня

Reply:
Вот: https://api.binance.com/api/v3/ticker/bookTicker
symbol: `XEMBNB`

Из за этого было деление на 0 = Infinity: https://app.bugsnag.com/alfa-genesis/kassa-dot-cc-1/errors/608847c25c51820007061393?filters[event.since][0]=30d&filters[error.status][0]=fixed

Reply:
Пришлось эту проверку добавить.

Reply:
Понял, спасибо! Давай тогда `rate['askPrice'].to_f.zero? || rate['bidPrice'].to_f.zero?`